.github/workflows/build.yml

```
@@ -0,0 +1,21 @@
+name: Build and Test
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
```

> Each major Go release is supported until there are two newer major releases. ([source](https://go.dev/doc/devel/release))

Shall we bump this to the latest?

```suggestion
        go-version: ^1.18
```

---

increased